dev-tools/packaging/templates/docker/docker-entrypoint.edot.tmpl

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Update umask to retain group write permissions on runtime directories: $root/tmp/default, $root/logs/default and $root/run/default
+umask 0007
+
+set -eo pipefail
+
+# For information on the possible environment variables that can be passed into the container. Run the following
+# command for information on the options that are available.
+#
+# `./elastic-agent container --help`
+#
+
+exec {{ .BeatName }} otel "$@"
```

Comment:
I think we can probably start using the `edot-colletor` image directly in our existing tests where ever we use ELASTIC_AGENT_OTEL: 

For example: https://github.com/michalpristas/elastic-agent/blob/29051e1bb7b5086beca0b453a0ac527dfeca7ead/testing/integration/kubernetes_agent_standalone_test.go#L207-L210

That will give you testing of the edot image. I would say we will eventually deprecate the ELASTIC_AGENT_OTEL variable in place of this container image rather than maintain it.

Let's add a log message that ELASTIC_AGENT_OTEL is deprecated and that you should use the edot-collector image directly instead so we can just delete it in 18 months or whatever time we consider reasonable since it will have been formally deprecated.

Reply:
The Helm charts can also be updated to use the edot-collector image directly https://github.com/michalpristas/elastic-agent/blob/29051e1bb7b5086beca0b453a0ac527dfeca7ead/deploy/helm/edot-collector/kube-stack/values.yaml#L36-L38